Security/Guard/JWTTokenAuthenticator.php

```diff
@@ -285,6 +286,14 @@ protected function loadUser(UserProviderInterface $userProvider, array $payload,
             return $userProvider->loadUserByUsernameAndPayload($identity, $payload);
         }
 
+        if ($userProvider instanceof ChainUserProvider) {
+            foreach ($userProvider->getProviders() as $provider) {
+                if ($provider instanceof PayloadAwareUserProviderInterface) {
```

Comment:
It's better to wrap this part into `try-catch` like it's done in original `Chain` provider https://github.com/symfony/security/blob/9b4060f040be91d55db67831bf83981097c609b1/Core/User/ChainUserProvider.php#L52

Reply:
@CvekCoding good catch. Would you mind sending a PR or shall I do it?

Reply:
Sure, I'll do this today.

Reply:
@chalasr when this fix can be released?

Reply:
@CvekCoding it has been released in https://github.com/lexik/LexikJWTAuthenticationBundle/releases/tag/v2.8.0

Reply:
Done https://github.com/lexik/LexikJWTAuthenticationBundle/pull/766